README.md

```diff
@@ -96,26 +96,23 @@ optional arguments:
                         is updated on the Adobe side so that the memberships
                         in mapped groups matches the customer side.
   --remove-nonexistent-users
-                        Causes the user sync tool to remove Federated users
-                        that exist on the Adobe side if they are not in the
-                        customer side AD. This has the effect of deleting the
-                        user account if that account is owned by the
-                        organization under which the sync operation is being
-                        run.
+                        Causes the user sync tool to remove users that exist on
+                        the Adobe side if they are not in the customer side AD.
```

Comment:
customer side directory.  Try and Find All on "AD" (case sensitive) and remove everyone you find! :)